app/views/homepage/index.html.erb

```diff
@@ -98,159 +98,6 @@
 
     <hr class="home-divider" id="more-on-govuk" aria-hidden="true" />
 
```

Comment:
Question on the `hr` and whether it's better to change these to `border-top`s now (or in `new-homepage`)

Reply:
I reckon personally let's do it in the full merge. That way we can see everything all at once and more effectively make decisions about the borders/spacing. If you'd really prefer I remove this now then I'll remove it now.

Reply:
No problem, agreed it makes more sense to revisit this after the full merge.